src/plugins/layout.ts

```diff
@@ -86,7 +108,15 @@ export default class Layout implements IPlugin {
   // Add node to the ShadowDom to store initial adjacent node info in a layout and obtain an index
   private discoverNode(node: Node): INodeInfo {
     let shadowNode = this.shadowDom.insertShadowNode(node, getNodeIndex(node.parentNode), getNodeIndex(node.nextSibling));
-    return shadowNode.computeInfo();
+    return this.computeInfo(shadowNode);
+  }
+
+  private computeInfo(shadowNode) {
```

Comment:
>private computeInfo(shadowNode) { [](start = 2, length = 33)

Let's not create an "extra content" wrapper around computeInfo - this would de facto make output of shadowNode.computeInfo() not trustworthy

Reply:
Agreed. While coding I came full circle and now extra content wrapper is no longer needed. Fixed it.

---
In reply to: [243075614](https://github.com/Microsoft/clarity-js/pull/116#discussion_r243075614) [](ancestors = 243075614)